custom_components/neakasa/translations/pl.json

```diff
@@ -0,0 +1,103 @@
+{
+    "title": "Neakasa",
+    "config": {
+        "step": {
+            "user": {
+                "title": "Konto Neakasa",
+                "description": "Dane logowanie zostaną użyte do pobrania danych z serwera.",
+                "data": {
+                    "username": "Nazwa użytkownika",
+                    "password": "Hasło"
+                }
+            },
+            "device": {
+                "description": "Wybierz jedno z dostępnych urządzeń.",
+                "data": {
+                    "device_id": "ID urządzenia"
+                }
+            }
+        },
+        "abort": {
+            "connenction": "Błąd połączenia.",
+            "authentication": "Błąd uwierzytelnienia.",
+            "unknown": "Nieznany błąd."
+        }
+    },
+    "entity": {
+        "button": {
+            "clean": {
+                "name": "Wyczyść"
+            },
+            "level": {
+                "name": "Wypoziomuj"
+            }
+        },
+        "binary_sensor": {
+            "bin_full": {
+                "name": "Pojemnik pełen"
+            }
+        },
+        "sensor": {
+            "sand_percent": {
+                "name": "Poziom żwirku"
+            },
+            "wifi_rssi": {
+                "name": "WiFi RSSI"
+            },
+            "stay_time": {
+                "name": "Długość ostatniego pobytu"
+            },
+            "last_usage": {
+                "name": "Ostatni pobyt"
+            },
+            "current_status": {
+                "name": "Status kuwety",
+                "state": {
+                    "idle": "Oczekiwanie",
+                    "cleaning": "Czyszczenie",
+                    "leveling": "Poziomowanie"
+                }
```

Comment:
please add translation for status "flipover"

Reply:
Done.